### DISCUSSION TITLE
Add 'Bengali' (bn) to locale.

### DISCUSSION THREAD

**Discussion by whoismaruf** [anuraghazra/github-readme-stats#1154] [Ideas]
I want to add **Bengali (বাংলা)** to the stats card. How can I?

**Comment by andreakaye** [comment]
🖤🖤🖤🖤🖤🖤🖤🖤

**Comment by anuraghazra** [comment]
Hi @whoismaruf 

You can go ahead and add bengali locale in this file
https://github.com/anuraghazra/github-readme-stats/blob/master/src/translations.js

**Reply by whoismaruf** [reply]
I did a pull request mate, can you authorize it? I can't wait to see Bengali on my profile 😑

**Comment by rickstaa** [comment]
For future reference, the pull request of @whoismaruf was merged as #1620.
